packages/api/src/entities/MinistryEntity.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Represents ministry entity.
+ */
+export default class MinistryEntity {
+    /**
+     * Creates a new instance of Ministry Entity class.
+     * @param sigle_ministere - The minestry acronyme.
+     * @param code_ministere - The ministry code.
+     * @param nom_ministere - The ministry name.
+     */
+    constructor(
+        /*
+        normalement tous ces attributes devraient être
```

Comment:
En principe, tu dois mettre `| null` car sinon ton code va s'attendre à ce qui y ai toujours une valeur et donc source de bug. Il ne va rien gérer du tout car le typescript est juste une couche syntaxique et donc la vérification des types ne se fait pas à l'exécution.

Reply:
ça marche! En gros il faut que je décide quand est-ce que l'attribut peut être optionnel ou quand est-ce que il doit être renseigné et dans ce dernier cas, il faudra mettre en place un validateur

Reply:
TO DO Giulia : add validator